toolkit/scripts/check_required_kernel_configs.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Script to check for required kernel configs in a kernel config file
+# Usage: python3 check_required_kernel_configs.py --required_configs <path to json of required configs> --config_file <path to config being checked>
+
+import json
+import argparse
+import sys
+from kernel_sources_analysis import get_data_from_config, extract_kernel_dir_name, extract_config_arch
+
+# Define a class
+    #incorrect configs is map: {config: (newValue, expectedValue, comment, PR)}
```

Comment:
Refactoring leftover? I'm not sure what this comment is about.

Reply:
Removed